ninja-core/src/main/java/ninja/session/Session.java

```diff
@@ -30,7 +30,8 @@
     String AUTHENTICITY_KEY = "___AT";
     String ID_KEY = "___ID";
     String TIMESTAMP_KEY = "___TS";
-    
+	String EXPIRY_TIME_KEY = "___EP";
```

Comment:
Ok!  The rest of that file is indented with tabs (below that line) though - didn't reformat it, as it would just confuse the change request.